docs/guides/BasicToolsUse.md

```
@@ -43,18 +43,18 @@ We will use the OpenAI GPT model since it natively supports tool use. Make sure
 ```python
 from draive import TextGeneration, load_env
-from draive.openai import OpenAIChatConfig, OpenAI
+from draive.openai import OpenAIResponsesConfig, OpenAI
 
 load_env()
 
 async with ctx.scope(
     "basics",
-    # define used LMM to be OpenAI within the context
-    OpenAIChatConfig(model="gpt-4o-mini"),
+    # initialize OpenAI client within the context
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)